docs/_includes/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,353 @@
+|{adp} |{adr} |Notes
+
+
+|`big`, `small`, `underline`, `overline`, `line-through`, colors
+|_deprecated_
+|Character attributes to apply formatting directly. 
+Usually better to apply a role, then apply the formatting based on that role by using a stylesheet.
+
+// old table says not, but...
+|`halign`, `valign` for table cells
+|Column and cell specifiers
+|See <<cell>>.
+
+|`infile`
+.2+|_not implemented_
+.2+|Provides the name and directory of the current document. 
+(Distinct from `docfile`, because `infile` may be an included document, and `docfile` is always the master document.)
+No {adr} equivalent.
+
+|`indir`
+
+|`asciidoc`
+|`asciidoctor` 
+|{adp} sets `asciidoc` to show that it is the current processor. 
+{adr} sets `asciidoctor` instead.
+
+// Ref migration guide
+|`toc2`
+.2+|`toc`
+.2+|Combined in a single attribute, see <<toc>>.
+
+// Ref migration guide
+|`toc-placement`
+
+// No checked
+|`options="pgwide"`
+|_not implemented_
+|DocBook attribute to make tables full page width, whatever the current indent.
+No {adr} equivalent.
+
+// Ref ap UG. I know it workls in ad tables; no ref to ad paragrphs so assume not implemented.
+|options="unbreakable"
+|
+|In {adr}, this only works for tables, not paragraphs.
+
+// http://www.methods.co.nz/asciidoc/userguide.html#X39
+// an ugly hack so not implemented
+|`plaintext`
+|_not implemented_
+|{adp} uses this to suppress inline substitutions and retain block indents when importing large blocks of plain text.
+{adr} deliberately does not implement it; the closest {adr} equivalent is a pass block.
+
+// old table
+|`replacements2`
+|`post_replacements`
+|Renamed.
+
+// does this need an explanation?
+|`presubs`
+|-
+|Not required.
+
+// rarely used so dont mention it
+//|`sgml`
+//|_not implemented_
+//|SGML is archaic and has been replaced by XML.
+
+// It may be a perversion, but it is a useful perversion!
+|`showcomments`
+|_not implemented_
+|In {adp}, turns single line comments into DocBook "remark" elements.
+{adr} considers this a bad use of comments. 
+If you want to pass remarks to the output, use an extension, or:
+----
+ifdef:showcomments[]
+<remark>Your comment here</remark>
+endif[]
+----
+
+|`specialwords`
+|_not implemented_
+|In {adp}, applies special formatting to named text.
+In {adr} this could be implemented using an extension.
+
+|`tabsize` (in-document and include directive)
+|in-document only
+|{adp} applies it to all text;
+{adr} only applies it to blocks with a verbatim content model (listing, literal, etc). Both have a default of 8. 
+For all other text, {adr} tabs are fixed at 4 spaces, see <<normalize-block-indentation>>.
+
+|====
+
+[[migrate-stylesheet]]
+=== Default HTML Stylesheet
+The {adp} and Acidoctor stylesheets are completely compatible, because their formatting is based on the same role attributes.
+If you happen to prefer the {adp} stylesheet, you can use it by copying it from the {adp} "stylesheets" directory and telling {adr} to use it with 
```

Comment:
When you refer to a path, you should write it as follows:

```
[.path]_stylesheets_
```

It will appear in italics for now, but gives us to option to style it differently in the future.